Sources/SWBUniversalPlatform/Specs/Clang LLVM 1.0.xcspec

```diff
@@ -2970,6 +2970,16 @@
                 };
             },
 
+            {
+                Name = "CLANG_ENABLE_COMPILE_CACHE";
+                Type = Enumeration;
+                Values = (
```

Comment:
Does this constrain the values or only impact help text? I'm not clear what it means for NOT_SET

Reply:
This is similar to `SWIFT_ENABLE_EXPLICIT_MODULES` [here](https://github.com/swiftlang/swift-build/blob/f67ead1006bedb2243a55b149184a65287899008/Sources/SWBUniversalPlatform/Specs/Swift.xcspec#L923).

Is there a concern @owenv ?

Reply:
Does the fact we are setting a new default value for the compiler-specific settings mean that NOT_SET will never be used for these anymore? If so maybe they can be changed to Boolean?

Reply:
It's been a while since I last looked at this, but I think it's safe to change this to a boolean setting since it's always going to get $(COMPILATION_CACHE_ENABLE_CACHING) as a backstop now

Reply:
I've changed `CLANG_ENABLE_COMPILE_CACHE` and `SWIFT_ENABLE_COMPILE_CACHE` to boolean macros.